compiler/optimizer/ValuePropagationTable.hpp

```diff
@@ -403,10 +403,10 @@ const ValuePropagationPtr constraintHandlers[] =
 
    constrainCmpeq,           // TR::acmpeq
    constrainCmpne,           // TR::acmpne
-   constrainCmplt,           // TR::acmplt
-   constrainCmpge,           // TR::acmpge
-   constrainCmpgt,           // TR::acmpgt
-   constrainCmple,           // TR::acmple
+   constrainCmp,             // TR::acmplt
+   constrainCmp,             // TR::acmpge
+   constrainCmp,             // TR::acmpgt
+   constrainCmp,             // TR::acmple
```

Comment:
Could you elaborate on why this seems to have changed from something more specific to something more general?